custom_components/tuya_local/translations/en.json

```diff
@@ -355,6 +355,42 @@
                     "celsius": "Celsius",
                     "fahrenheit": "Fahrenheit"
                 }
+            },
+            "oven_local_recipe": {
+                "name": "Local Recipe",
```

Comment:
I think "Built-in" is a closer translation of the Chinese than "Local".
Also, Home Assistant convention is for "recipe" in lower case.

Reply:
Indeed, I also believe using “Built-in” is more appropriate. I have done the same in the Chinese context.